packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactHost.kt

```diff
@@ -48,6 +48,8 @@ public interface ReactHost {
   /** Routes memory pressure events to interested components */
   public val memoryPressureRouter: MemoryPressureRouter
 
+  public fun isEdgeToEdgeEnabled(): Boolean
```

Comment:
I have some concern with adding this field to ReactHost as this more of a UI configuration that will eventually go away as edge-to-edge will become the default for Android and as we raise the minSdk. Doesn't seem like it is something we should keep in ReactHost.

Reply:
I agree, just tried to copy the current code structure, but it's not optimal. What would you think about removing all of this and just have an `EdgeToEdgeUtils.kt` somewhere?

```kotlin
object EdgeToEdgeUtils {
  fun isEnabled(): Boolean {
    return BuildConfig.IS_EDGE_TO_EDGE_ENABLED
  }
}
```